retroshare-gui/src/gui/TheWire/WireGroupExtra.cpp

```diff
@@ -34,21 +34,26 @@ WireGroupExtra::~WireGroupExtra()
 
 void WireGroupExtra::setUp()
 {
-    connect(ui.pushButton_masthead, SIGNAL(clicked() ), this , SLOT(addMasthead()));
-}
+	connect(ui.pushButton_masthead, SIGNAL(clicked() ), this , SLOT(addMasthead()));
+
+	int desired_height = ui.pushButton_masthead->height() + ui.removeButton->height() + ui.lineEdit_Tagline->height();
+	int desired_width =  3/1.0 * desired_height + ui.lineEdit_Tagline->width(); 
 
+	ui.label_masthead->setFixedSize(desired_width, desired_height);
+
+	setMasthead(QPixmap());
+}
 
 void WireGroupExtra::addMasthead()
 {
-    QPixmap img = misc::getOpenThumbnailedPicture(this, tr("Load Masthead"), 400, 100);
+    QPixmap img = misc::getOpenThumbnailedPicture(this, tr("Load Masthead"), 800, 600);

```

Comment:
absolute sizes usually don't work well. I do not know how to do better here anyway.

Reply:
i changed to this, on loading the orginal image get very blurred before crop.